src/pages/notes/components/NoteEditor.js

```diff
@@ -19,8 +20,10 @@ export const NoteEditor = (props) => {
     const [optionButton, setOptionButton] = useState(false);
     const [show, setShow] = useState(false);
     const [link, setLink] = useState('');
+    const [fullScreenFlag, setFullScreenFlag] = useState(false);
+    const {addToast} = useToast();
 
-    const {editorState, onChange, readOnly} = props;
+    const {editorState, onChange, readOnly, editorRef} = props;
```

Comment:
onChange is a JSX event listener name, preferably make the name of the function more descriptive. But !important.

Reply:
Yes.
onChange is a common name, but it has been widely used here in multiple scenarios.
Updating that would require plenty of work.
Postponing this for now.

Reply:
Need to improve naming conventions. Names need to be more descriptive now onwards.